packages/createrepo_c/createrepo_c.spec

```
@@ -166,7 +169,7 @@ pushd build-py3
       -DWITH_LEGACY_HASHES=%{?with_legacy_hashes:ON}%{!?with_legacy_hashes:OFF} \
       -DWITH_ZSTD=%{?with_zstd:ON}%{!?with_zstd:OFF} \
       -DENABLE_DRPM=%{?with_drpm:ON}%{!?with_drpm:OFF} \
-      -DPYTHON_EXECUTABLE=/usr/bin/python3.9 -DPYTHON_LIBRARY=/usr/lib64/libpython3.9.so
+      -DPYTHON_EXECUTABLE=/usr/bin/python3.11 -DPYTHON_LIBRARY=/usr/lib64/libpython3.11.so
```

```suggestion
      -DPYTHON_EXECUTABLE=%{__python3} -DPYTHON_LIBRARY=/usr/lib64/libpython3.11.so
```

I also looked for a macro for the lib, but couldn't find a good one. This is also a weird one:
```
>>> import sys ;sys.version_info
sys.version_info(major=3, minor=11, micro=5, releaselevel='final', serial=0)
>>> sys.stdout.write('{0.major}.{0.minor}.'.format(sys.version_info))
3.11.5
>>> sys.stdout.write('{0.major}.{0.minor}'.format(sys.version_info))
3.114
```

This comes from:
```
%python3_version %(RPM_BUILD_ROOT= %{__python3} -Ic "import sys; sys.stdout.write('{0.major}.{0.minor}'.format(sys.version_info))")
```

I have no idea why it somehow adds the micro version.

---

Ok, scratch build using `%{__python3}` worked 

https://copr.fedorainfracloud.org/coprs/g/theforeman/pulpcore-nightly-staging-scratch-2cdd8425-0950-5979-bea5-035f5b6c9072/build/6629861/ 

Commit Added